Client/Frontend/Settings/AppSettingsOptions.swift

```diff
@@ -519,6 +519,34 @@ class SentryIDSetting: HiddenSetting {
     }
 }
 
+class ToggleOnboarding: HiddenSetting {
+    override var title: NSAttributedString? {
+        return NSAttributedString(string: NSLocalizedString("Debug: Toggle onboarding type", comment: "Debug option"), attributes: [NSAttributedString.Key.foregroundColor: UIColor.theme.tableView.rowText])
+    }
+
+    override func onClick(_ navigationController: UINavigationController?) {
+        let onboardingResearch = OnboardingUserResearch()
+        let type = onboardingResearch.onboardingScreenType
+        var newOnboardingType: OnboardingScreenType = .versionV2
```

Comment:
`let newOnboardingType = type ==.versionV2 ?? .versionV1 : .versionV1` ?

Reply:
That coalescing operator ?? would only return .versionV2 and doesn't go to .versionV1 ...

I tried something like the one below. It works but had doubts about its readability

`
let newOnboardingType: OnboardingScreenType = type == nil ? .versionV1 : type == .versionV2 ? .versionV1 : .versionV2`

Thoughts? 🤔

Reply:
Older version
//        var newOnboardingType: OnboardingScreenType = .versionV2
//        if type == nil {
//            newOnboardingType = .versionV1
//        } else if type == .versionV2 {
//            newOnboardingType = .versionV1
//        }

Reply:
oops, didn't realize that, keep as is!